src/main/java/redis/clients/jedis/resps/KeyedListElement.java

```diff
@@ -0,0 +1,45 @@
+package redis.clients.jedis.resps;
+
+import redis.clients.jedis.util.SafeEncoder;
+
+public class KeyedListElement {
```

Comment:
@sazzad16 I think some JavaDoc that will describe this class is needed

Reply:
@gkorland JavaDoc added.